kids/lib/components/player/player_view.dart

```diff
@@ -141,6 +164,26 @@ class PlayerView extends HookConsumerWidget {
       };
     });
 
+    // Autoplay
+    useEffect(() {
+      void onEnded(PlaybackEndedEvent event) {
+        if (hasAutoplayNext && !isAutoplayNextCancelled.value && episode != null) {
+          navigateToEpisode(episode!.next.first);
```

Comment:
.first in general is scary in dart because it means: give me the first or throw an error.
You either want:
- .firstOrNull
- or a `if (episode.next.isNotEmpty)`